docker/conf/docker-compose-mac.yml.dist

```diff
@@ -1,4 +1,11 @@
 version: "3.4"
+
+x-variables:
+    common_build_args: &common_build_args
+        www_data_uid: 1000
+        www_data_gid: 1000
```

Comment:
Default values for Mac should be `501:20`

Reply:
(should be in `fixup! monorepo docker-compose.yml files now has common_build_args extracted into extension field`)